scripts/check-queries.lua

```diff
@@ -35,7 +35,7 @@ local function do_check()
   local captures = extract_captures()
   local last_error
 
-  print "::group::Check parsers"
+  io.write "::group::Check parsers\n"
```

Comment:
maybe add a local wrapper around that idiom?

Reply:
but would be also fine with this solution.

Reply:
> maybe add a local wrapper around that idiom?

Sorry, I'm still new to lua. What do you mean? Can you give an example/link?

Reply:
write your own `print` that uses `io.write`. Like `local function io_print(text) io.write(text .. '\n') end`